coffeecard/CoffeeCard.Library/Services/v2/WebhookService.cs

```diff
@@ -86,16 +88,6 @@ private async Task SyncWebhook(WebhookConfiguration webhook)
                     await DisableAndRegisterNewWebhook(webhook);
                     return;
                 }
-
```

Comment:
Maybe I'm missing something, but isn't this the bit that's supposed be feature toggled?